### PR TITLE
Completed BETA for KavSim v2.5.1

### PR DESCRIPTION
**Pull request by Jak-Kav**
This PR contains the completed BETA project for the Kav Simulations functionality in the MF BETA v10.0.x.x

All libraries have been bug fixed, and there is now a complete working example mcc and mfmc directory of files for all displays.

Ideally, this would be pushed into the next MF BETA update :-)

### CODE REVIEW

File: KAV_Simulation/KAV_A3XX_RAD_TCAS_LCD.cpp [hunk 1]
```diff
@@ -126,18 +126,29 @@ void KAV_A3XX_RAD_TCAS_LCD::setAllDots(bool enabled)
...
-    value = value / 10;
-    displayDigit(DIGIT_FIVE, (value % 10));
-    value = value / 10;
-    displayDigit(DIGIT_FOUR, (value % 10));
-    value = value / 10;
-    displayDigit(DIGIT_THREE, (value % 10));
-    value = value / 10;
-    displayDigit(DIGIT_TWO, (value % 10));
-    displayDigit(DIGIT_ONE, (value / 10));
+    // TODO: Add a condition that if 0, then show 'data' on the display, then break.
```
Comment (elral) [inline KAV_Simulation/KAV_A3XX_RAD_TCAS_LCD.cpp:129]: Is this still a TODO? Looks like it is done, so comment could be deleted

File: KAV_Simulation/KAV_A3XX_RUDDER_LCD.cpp [hunk 2]
```diff
@@ -212,7 +241,8 @@ void KAV_A3XX_RUDDER_LCD::displayDigit(uint8_t address, uint8_t digit)
  */
 void KAV_A3XX_RUDDER_LCD::set(int16_t messageID, char *setPoint)
 {
-    int32_t data = atoi(setPoint);
+    // Using `strtol()` instead of `atoi()` to allow for negative numbers.
```
Comment (elral) [inline KAV_Simulation/KAV_A3XX_RUDDER_LCD.cpp:244]: atoi() returns negative numbers too, but it is limited to 16bit for AVR's (like you comment in KAV_A3XX_FCU_LCD::set() )

File: KAV_Simulation/KAV_A3XX_RAD_TCAS_LCD.cpp (hunk 1 above)
Comment (Jak-Kav) [reply KAV_Simulation/KAV_A3XX_RAD_TCAS_LCD.cpp]: Updated

File: KAV_Simulation/KAV_A3XX_RUDDER_LCD.cpp (hunk 2 above)
Comment (Jak-Kav) [reply KAV_Simulation/KAV_A3XX_RUDDER_LCD.cpp]: Updated.

**Pull request merged**